TextToSpeech/impl/TTSSpeaker.cpp

```diff
@@ -222,6 +263,11 @@ bool TTSConfiguration::isValid() {
     return true;
 }
 
+bool TTSConfiguration::isSwitchRequired()
```

Comment:
This function name is misleading. From what I understood from the implementation it looks like it is used in the context of "whether the local endpoint is configured", renaming it to "hasValidLocalEndpoint()" or similar would be more readable